source/components-and-apis.rst

```diff
@@ -106,6 +106,10 @@ Components
     - `GitHub - Kafka2Ftp`_
     -
     -
+  * - Kafka Websocket
```

Comment:
I think you can change  "Kafka Websocket" to "Kafka WS (Kafka Websocket)" or just "Kafka WS"